client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationStatusColumn/InvestigationStatusColumn.tsx

```diff
@@ -11,10 +11,11 @@ import useCustomSwal from 'commons/CustomSwal/useCustomSwal';
 import InvestigationMainStatus from 'models/enums/InvestigationMainStatus';
 
 import useStyles from './InvestigationStatusColumnStyles';
+import { transferedSubStatus } from '../useInvestigationTable';
 
 const InvestigationStatusColumn = (props: Props) => {
 
-    const { investigationStatus, investigationSubStatus, epidemiologyNumber, moveToTheInvestigationForm, statusReason, } = props;
+    const { investigationStatus, investigationSubStatus, epidemiologyNumber, moveToTheInvestigationForm, statusReason, transfered } = props;
```

Comment:
In my opinion transfered should be assigned a more meaningful name such as wasInvestigationTransfered